### PR TITLE
Execute code 

### PR DESCRIPTION
**Pull request by leirbag95**
Upload / Write code documentions

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -113,6 +113,10 @@ nav:
       - 'Index': computing/index.md
       - Custom Domain:
           - 'Add a Custom Domain': computing/custom_domain/setup.md
+      - Execute Your Code:
+          - 'Index': computing/code_to_execute/index.md
```
Comment (hoh) [inline mkdocs.yml:117]:
```suggestion
          - 'Introduction': computing/code_to_execute/index.md
```

File: docs/computing/code_to_execute/write_your_code.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+When writing your code directly in our IDE, there's one important requirement you need to follow:
+
+### The `app` Function
+
+Your code should contain an `app` function. This function acts as the entry point to your program, meaning it's where the execution of your code begins. The `app` function is crucial for our platform to correctly execute and interact with your code. Here's what you need to keep in mind:
+
+- **For Python**: Define an `app` function within any of your Python files. This function will be the starting point of your application.
+  
+  ```python
```
Comment (hoh) [inline docs/computing/code_to_execute/write_your_code.md:23]: This tag is not closed and does not work.

image

**Review by hoh — COMMENTED** [review]
Thanks for starting this doc. 

Should this go in a section `Tools > Console` ?

**Review by hoh — COMMENTED** [review]
Thanks for these additions !

I would like to discuss two points regarding the language used:

1. The use of `we` is not consistent with the rest of the documentation. The rest of the documentation speaks about "the platform", "aleph.im", ... but not "we" or "us".
2. Some sentences include superlative that do not bring anything to the reader, such as _"You will be able to easily integrate your projects and get them running smoothly."_

Can we discuss how to ensure that the documentation is consistent regarding the language used ?

**Comment by leirbag95** [comment]
> Thanks for these additions !
> 
> I would like to discuss two points regarding the language used:
> 
> 1. The use of `we` is not consistent with the rest of the documentation. The rest of the documentation speaks about "the platform", "aleph.im", ... but not "we" or "us".
> 2. Some sentences include superlative that do not bring anything to the reader, such as _"You will be able to easily integrate your projects and get them running smoothly."_
> 
> Can we discuss how to ensure that the documentation is consistent regarding the language used ?

I've made adjustments to the text by removing references to the second-person plural, opting instead for a broader, more impersonal style. Additionally, I've cut out extraneous content that didn't add value, streamlining the message for better clarity and focus.

**Comment by MHHukiewitz** [comment]
This conversation reads like it has been written by ChatGPT :'D

**Comment by MHHukiewitz** [comment]
I'd rename it to "Web Console", so it is clear that not the CLI is being meant here

**Comment by leirbag95** [comment]
> This conversation reads like it has been written by ChatGPT :'D

I take it as a compliment ^^

**Comment by MHHukiewitz** [comment]
Just discussing the Node.js example with @philogicae, as it does not yet seem to work as described in the docs. Should we postpone merging this until we clear up the situation? Or split it out into its own PR so we can describe the Python process for now?

**Comment by philogicae** [comment]
> Just discussing the Node.js example with @philogicae, as it does not yet seem to work as described in the docs. Should we postpone merging this until we clear up the situation? Or split it out into its own PR so we can describe the Python process for now?

Yes, this is not ready yet. If users are actually following the docs, it will not work. I will speak with @leirbag95 to fix it

**Comment by philogicae** [comment]
LGTM, we can merge.
Fixing JS functions is still in WIP: There is an incorrect linking of package dependencies. They should be uploaded and referenced as an immutable volume and mapped on /opt/node_modules. ATM, users are required to include the node_modules folder inside its code folder. We will have to update the docs again later when the fix will be deployed and tested.

**Pull request merged**